src/app/components/navbar/navbar.component.html

```diff
@@ -41,30 +79,59 @@
 				<div aria-labelledby="navbarDropdownMenuLink-6"
 						class="dropdown-menu dropdown-menu-lg-right dropdown-secondary">
 						<a class="dropdown-item waves-effect waves-light eborder-top" routerLink="/payment/pin" routerLinkActive="active"> Card & PIN</a>
-						<a class="dropdown-item waves-effect waves-light" routerLink="">International Card</a>
+						<a class="dropdown-item waves-effect waves-light" routerLink="/payment/international">International Card</a>
 						<a class="dropdown-item waves-effect waves-light" routerLink="/payment/saved-card">Saved Card</a>
 				</div>
 			</li>
 
       <li *ngIf="authenticated" class="nav-item avatar dropdown">
-        <a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle waves-effect waves-light"
-          data-toggle="dropdown" id="navbarDropdownMenuLink-6">
+        <a
+          aria-expanded="false"
+          aria-haspopup="true"
+          class="nav-link dropdown-toggle waves-effect waves-light"
+          data-toggle="dropdown"
+          id="navbarDropdownMenuLink-6"
+        >
           <span class="profile-ava">
-            <img [src]="profileImage" alt="avatar image" class="rounded-circle z-depth-0" height="35">
+            <img
```

Comment:
Hey @patrickf949, we need to add a descriptive alt text to the images for when they cant display.

Reply:
Yes @CryceTruly , thats great, please suggest what may be more convenient since there are actually multiple use cases of images in the application, though for this feature currently I haven't worked with any